webapp/pages/settings/index.vue

```diff
@@ -135,15 +135,12 @@ export default {
             })
           }
         })
-        .then(data => {
-          this.$toast.success('Updated user')
-        })
-        .catch(err => {
-          this.$toast.error(err.message)
-        })
-        .finally(() => {
-          this.sending = false
-        })
+        this.$toast.success(this.$t('settings.data.success'))
+      } catch (err) {
+        this.$toast.error(err.message)
+      } finally {
+        this.loadingData = false
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-17T14:08:59Z" title="Wednesday, April 17th 2019, 4:08:59 pm +02:00">Apr 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -135,15 +135,12 @@ export default {
             })
           }
         })
-        .then(data => {
-          this.$toast.success('Updated user')
-        })
-        .catch(err => {
-          this.$toast.error(err.message)
-        })
-        .finally(() => {
-          this.sending = false
-        })
+        this.$toast.success(this.$t('settings.data.success'))
+      } catch (err) {
+        this.$toast.error(err.message)
+      } finally {
+        this.loadingData = false
```


```suggestion
        this.pending = false
```
I believe `pending` is already a convention in this project.

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-19T10:01:31Z" title="Friday, April 19th 2019, 12:01:31 pm +02:00">Apr 19, 2019</time>_

---


![image](https://user-images.githubusercontent.com/1238238/56419523-c587ae80-629a-11e9-936f-f3ea310ce267.png)

Operation name on ds-button template is `loading`

Suggestion: No Triage

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-19T11:51:27Z" title="Friday, April 19th 2019, 1:51:27 pm +02:00">Apr 19, 2019</time>_

---


Yes, `loading` is good if the component expects a property with that name.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-19T11:52:10Z" title="Friday, April 19th 2019, 1:52:10 pm +02:00">Apr 19, 2019</time>_

---


What do you mean by

> Suggestion: No Triage

by the way?

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-20T11:31:25Z" title="Saturday, April 20th 2019, 1:31:25 pm +02:00">Apr 20, 2019</time>_

---


`no triage` means that there will be no fix for the raised issue. It is a medical term and would translate to smth like `keine Behandlung` (German). It is used with google repos.
If I `Suggestion: No Triage`, I present an argument why I don't want to fix this and ask for approval of this course of action.